### ISSUE TITLE
Make a point of explaining why this is a feature that can't be restricted to asm.js

### ISSUE THREAD

**Issue by lars-t-hansen** [tc39/ecmascript_sharedmem#33]
This came up several times in the TC39 meeting.  It will be useful to capture some of the points of that discussion in the DISCUSSION document or perhaps in the introduction to the spec.

**Comment by taisel** [comment]
For one thing it already provided a benefit in real world JavaScript.

I added it to https://github.com/taisel/IodineGBA recently and it was specifically used not for performance reasons, but to free up the message queue on the main thread. By freeing that up, the FPS is more stable. It also allows for non hardware accelerated browsers to be slow on drawing canvas and not tie up the emulation core as much.

Files it's implemented in:
https://github.com/taisel/IodineGBA/blob/master/user_scripts/IodineGBAWorkerGlueCode.js
https://github.com/taisel/IodineGBA/blob/master/user_scripts/IodineGBAWorkerGlueCodeWorker.js

At least we got our real world use case. :)
